src/table/styling-utils.js

```diff
@@ -15,10 +15,32 @@ export function getHeadStyle(layout, theme) {
 
 export function getBodyStyle(layout, theme) {
   const content = layout.components?.[0]?.content;
+  //  "hoverFontColor": {
+  //   "index": -1,
+  //   "color": null
+  // } means the hover fornt color is unset.
+  const unsetHoverFontColor =
+    !content.hoverFontColor || JSON.stringify(content.hoverFontColor) === JSON.stringify({ index: -1, color: null });
+  const unsetHoverColor =
+    !content.hoverColor || JSON.stringify(content.hoverColor) === JSON.stringify({ index: -1, color: null });
+  const unsetProps = unsetHoverFontColor && unsetHoverColor;
+
+  // 1. hoverEffect is true, there is no font color (hover or non-hover) and a hover background color, the hover font color becomes white and the hover background color is set.
```

Comment:
Close, but not quite. If the hover background color is "dark", we set the font color to white, otherwise it is ontouched